Source/Scene/Camera.js

```diff
@@ -361,6 +362,17 @@ Camera.prototype._updateCameraChanged = function () {
 
   var percentageChanged = camera.percentageChanged;
 
+  if (!defined(camera._changedHeading)) {
+    camera._changedHeading = camera.heading;
+  }
+
+  var headingChanged = camera._changedHeading === camera.heading ? false : true;
+
+  if (headingChanged) {
+    camera._changed.raiseEvent("Camera Heading Changed");
```

Comment:
The event listener expects a number (percentage) instead of a string. Even though the heading change event doesn't require a minimum percentage threshold you should still compute a percentage and pass that through.

This is causing a crash in http://localhost:8080/Apps/Sandcastle/index.html?src=Camera.html when you select "Camera changed event" and change the heading of the camera.

Reply:
I was able to update the `Camera.js` code. My updates fixed the following two spec:
- `Scene/Scene raises the camera changed event on direction changed`
- `Scene/Scene raises the camera changed event on position changed`

However, there is still an issue with `Scene/Scene raises the camera changed event in 2D`. It seems like the heading values generated in 2D are somewhat extraneous. There are two simple workarounds that I can think of:
1. Update `Camera.js` line `377` to input an artificial value instead of the error-causing and somewhat irrelevant calculated value.
2. Modify the `Scene/Scene raises the camera changed event in 2D` to not expect `args[0][0]` to be greater than `s.camera.percentageChanged`.

What are your thoughts?

Reply:
I believe that this is resolved in my most recent commit - I reverted `Scene/Scene raises the camera changed event in 2D` back to the original spec.